server/grpc/server.go

```diff
@@ -0,0 +1,47 @@
+package grpc
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+
+	"github.com/cosmos/cosmos-sdk/server/types"
+)
+
+const (
+	// GRPCBlockHeightHeader is the gRPC header for block height
+	GRPCBlockHeightHeader = "sdk-block-height"
+)
+
+// StartGRPCServer starts a gRPC server on the given address.
+func StartGRPCServer(app types.Application, address string) (*grpc.Server, error) {
+	grpcSrv := grpc.NewServer()
+	app.RegisterGRPCServer(grpcSrv)
+
+	// Reflection allows external clients to see what services and methods
+	// the gRPC server exposes.
+	reflection.Register(grpcSrv)
```

Comment:
should we add a test that reflection works?

Reply:
done ✅